### PR TITLE
DOCSP-31324: keepalive settings removed + defaults

### PR DESCRIPTION
**Pull request by rustagir**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-node/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-31324>
**Staging**
- [faq entries](https://docs-mongodbcom-staging.corp.mongodb.com/node/docsworker-xlarge/DOCSP-31324-tcp-keepalive-defaults/faq/#what-does-the-keepalive-option-do-)
- [what's new](https://docs-mongodbcom-staging.corp.mongodb.com/node/docsworker-xlarge/DOCSP-31324-tcp-keepalive-defaults/whats-new/#what-s-new-in-6.0)
- [upgrade guide](https://docs-mongodbcom-staging.corp.mongodb.com/node/docsworker-xlarge/DOCSP-31324-tcp-keepalive-defaults/upgrade/#version-6.x-breaking-changes)

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?

### CODE REVIEW

File: source/upgrade.txt [hunk 1]
```diff
@@ -70,6 +70,10 @@ Version 6.x Breaking Changes
   ``tlsCertificateKeyFile`` connection options when you call the
   ``MongoClient.connect()`` method, not when you create the
   ``MongoClient`` instance.
+- The driver removes the ``keepAlive`` and ``keepAliveInitialDelay`` connection
+  options. Internally, the value of ``keepAlive`` is set to ``true`` and the
```
[the remark this answers is not in the document]
Comment (mongoKart) [reply source/upgrade.txt:74]: Also applies below

**Review by mongoKart — CHANGES_REQUESTED** [review]
Not any major blockers, but I'd appreciate another look

File: source/faq.txt [hunk 2]
```diff
@@ -238,32 +238,51 @@ to an operation that returns a ``Cursor``:
...
 
-If you experience unexpected network behavior, here
-are some things to check:
+You might experience unexpected network behavior if the firewall between
+your application and MongoDB is misconfigured. These firewalls can be
+overly aggressive in their removal of connections, which can lead to
+unexpected errors.
+
+To check if the firewall might be causing connection issues, confirm
+that it exhibits the following behavior:
```
[the remark this answers is not in the document]
Comment (rustagir) [reply source/faq.txt:275]: changed and changed the bullet point text to match

**Review by mongoKart — APPROVED** [review]
LGTM!

**Pull request merged**